lib/fastlane/plugin/dynatrace/helper/dynatrace_symlink_helper.rb

```diff
@@ -0,0 +1,76 @@
+module Fastlane
+  UI = FastlaneCore::UI unless Fastlane.const_defined?("UI")
+
+  module Helper
+    class DynatraceSymlinkHelper
+      @symlink_pattern = "*LLDB.framework"
+
+      def self.path_exists?(path)
+        unless path.nil?
+          return Dir.exist?(path) || File.exist?(path)
+        end
+        return false
+      end
+
+      def self.symlink_custom_lldb(lldb_path, destination_path)
+        require_path(destination_path)
+        require_path(lldb_path)
+        UI.message "Preparing to symlink custom LLDB framework path to: #{destination_path}"
+        delete_existing_lldb_symlinks(destination_path)
+        symlink(lldb_path, destination_path)
+      end
+
+      def self.auto_symlink_lldb(destination_path)
+        require_path(destination_path)
+        UI.message "Preparing to automatically symlink LLDB framework path to: #{destination_path}"
+        current_xcode_path = %x(xcrun xcode-select --print-path).chomp
+        active_lldb_path = active_lldb_path(current_xcode_path)
+        unless active_lldb_path.nil?
+          UI.message "LLDB framework found at: #{active_lldb_path}"
+          delete_existing_lldb_symlinks(destination_path)
```

Comment:
I'd move the deletion to dynatrace_action.rb#L65 so it's deleted on each run and recreated as necessary. If we have a symlink in place, but run the plugin without custom path and no auto link it won't get deleted.

Reply:
do you mean we should always delete the existing symlink on each run?

Reply:
Would be the simplest solution I think.